starters/next-react-query-tailwind/src/components/Counter/Counter.tsx

```diff
@@ -1,15 +1,16 @@
 import { useState } from 'react';
 
+const DEFAUT_COUNT = 0;
+
 export function Counter() {
   const [count, setCount] = useState(0);
```

Comment:
If you're setting `DEFAUT_COUNT`, it should also be the default for `useState` here.